src/Apps/Auction/Components/RegistrationForm.tsx

```diff
@@ -1,8 +1,8 @@
-import { Box, Button, Flex, Input, Sans, Serif } from "@artsy/palette"
+import { Box, Button, Flex, Sans, Serif } from "@artsy/palette"
+import { Address, AddressForm } from "Apps/Order/Components/AddressForm"
```

Comment:
The `AddressForm` should move out of `src/Apps/Order` and into `src/Components`. I can tackle that in a separate PR.